### PR TITLE
Add org.linuxshowplayer.LinuxShowPlayer 

### PR DESCRIPTION
**Pull request by FrancescoCeruti**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I have [built][build] and tested the submission locally.
- [ ] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub.
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

### non-standard permissions

Jack over PipeWire
```
"--system-talk-name=org.freedesktop.RealtimeKit1",
"--filesystem=xdg-run/pipewire-0"
```

Need access to actual file paths, to save relative paths in the application session
```
"--filesystem=home"
```

MIDI
```
"--device=all"
```

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission

### CODE REVIEW

File: org.linuxshowplayer.LinuxShowPlayer.json [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+    "sdk": "org.kde.Sdk",
+    "command": "linux-show-player",
+    "rename-icon": "linuxshowplayer",
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--socket=wayland",
+        "--socket=fallback-x11",
+        "--socket=pulseaudio",
+        "--system-talk-name=org.freedesktop.RealtimeKit1",
```
Comment (hfiguiere) [inline org.linuxshowplayer.LinuxShowPlayer.json:14]:
```suggestion
```

this has never worked and is unecessary

File: org.linuxshowplayer.LinuxShowPlayer.json [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+    ],
+    "cleanup": [
+        "*.la",
+        "*.a",
+        "/include",
+        "/lib/pkgconfig"
+    ],
+    "modules": [
+        {
+            "name": "Jack",
```
Comment (hfiguiere) [inline org.linuxshowplayer.LinuxShowPlayer.json:27]: you definitely do NOT need jack.

File: org.linuxshowplayer.LinuxShowPlayer.json [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://files.pythonhosted.org/packages/6d/13/57481471e0535151686ce18bb7e74a4884519474408872ffaca7732f9946/python_rtmidi-1.5.6.tar.gz",
+                    "sha256": "b2a0a399bb4a5e1a56477798afd41d022a18b5e954f6d0ff92b45bb99bee3710"
+                }
+            ]
+        },
+        {
+            "name": "liblo",
```
Comment (hfiguiere) [inline org.linuxshowplayer.LinuxShowPlayer.json:73]: please use liblo from the shared-modules.

File: org.linuxshowplayer.LinuxShowPlayer.json [hunk 4]
```diff
@@ -0,0 +1,226 @@
...
+            ],
+            "sources": [
+                {
+                    "type": "file",
+                    "url": "https://files.pythonhosted.org/packages/ec/1a/610693ac4ee14fcdf2d9bf3c493370e4f2ef7ae2e19217d7a237ff42367d/packaging-23.2-py3-none-any.whl",
+                    "sha256": "8c491190033a9af7e1d931d0b5dacc2ef47509b34dd0de67ed209b5203fc88c7"
+                }
+            ]
+        },
+        "pyqt-modules.json",
```
Comment (hfiguiere) [inline org.linuxshowplayer.LinuxShowPlayer.json:204]: there is a base app to use pyqt. Please use that.

File: org.linuxshowplayer.LinuxShowPlayer.json [hunk 5]
```diff
@@ -0,0 +1,226 @@
+{
+    "app-id": "org.linuxshowplayer.LinuxShowPlayer",
```
Comment (hfiguiere) [inline org.linuxshowplayer.LinuxShowPlayer.json:2]: the domain `linuxshowplayer.com` doesn't seem to exist, therefor the app-id doesn't meet the requirements.

Comment (FrancescoCeruti) [reply org.linuxshowplayer.LinuxShowPlayer.json:2]: It's `linuxshowplayer.org`

File: org.linuxshowplayer.LinuxShowPlayer.json [hunk 6]
```diff
@@ -0,0 +1,202 @@
...
+        "--device=all"
+    ],
+    "cleanup-commands": [
+        "/app/cleanup-BaseApp.sh"
+    ],
+    "cleanup": [
+        "*.la",
+        "*.a",
+        "/include",
+        "/lib/pkgconfig"
```
Comment (bbhtt) [inline org.linuxshowplayer.LinuxShowPlayer.json:27]: Add /share/man

File: org.linuxshowplayer.LinuxShowPlayer.json [hunk 7]
```diff
@@ -0,0 +1,202 @@
...
+        {
+            "name": "python-alsa",
+            "buildsystem": "simple",
+            "build-commands": [
+                "pip3 install --no-index --no-build-isolation --prefix=\"${FLATPAK_DEST}\" ."
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "ftp://ftp.alsa-project.org/pub/pyalsa/pyalsa-1.2.7.tar.bz2",
```
Comment (bbhtt) [inline org.linuxshowplayer.LinuxShowPlayer.json:144]: This has an https url.

**Comment by FrancescoCeruti** [comment]
bot, build org.linuxshowplayer.LinuxShowPlayer

**Comment by barthalion** [comment]
bot, build org.linuxshowplayer.LinuxShowPlayer

**Pull request closed without merging**